addons/html_builder/static/src/plugins/button_option.js

```diff
@@ -0,0 +1,112 @@
+            editingElement.after(" ");
+            // It is first the next button that we put in this variable because
+            // we want to copy as a priority the style of the previous button
+            // if it exists.
+            siblingButtonEl = nextSiblingEl;
+        }
+        if (previousSiblingEl?.matches(".btn")) {
+            previousSiblingEl.classList.add("mb-2");
+            editingElement.before(" ");
+            siblingButtonEl = previousSiblingEl;
+        }
+        if (siblingButtonEl) {
+            editingElement.classList.add("mb-2");
+        }
+        if (adaptAppearance) {
+            if (siblingButtonEl && !editingElement.matches(".s_custom_button")) {
+                // If the dropped button is not a custom button then we adjust
+                // its appearance to match its sibling.
+                if (siblingButtonEl.classList.contains("btn-secondary")) {
+                    editingElement.classList.remove("btn-primary");
+                    editingElement.classList.add("btn-secondary");
+                }
+                if (siblingButtonEl.classList.contains("btn-sm")) {
+                    editingElement.classList.add("btn-sm");
+                } else if (siblingButtonEl.classList.contains("btn-lg")) {
+                    editingElement.classList.add("btn-lg");
+                }
+            } else {
+                // To align with the editor's behavior, we need to enclose the
+                // button in a <p> tag if it's not dropped within a <p> tag. We only
+                // put the dropped button in a <p> if it's not next to another
+                // button, because some snippets have buttons that aren't inside a
+                // <p> (e.g. s_text_cover).
+                // TODO: this definitely needs to be fixed at web_editor level.
+                // Nothing should prevent adding buttons outside of a paragraph.
+                const btnContainerEl = editingElement.closest("p");
+                if (!btnContainerEl) {
+                    const paragraphEl = document.createElement("p");
+                    editingElement.parentNode.insertBefore(paragraphEl, editingElement);
+                    paragraphEl.appendChild(editingElement);
+                }
+            }
+            editingElement.classList.remove("s_custom_button");
+        }
+    }
+}
+
+registry.category("website-plugins").add(ButtonOptionPlugin.id, ButtonOptionPlugin);
+
+export class ButtonOption extends Component {
+    static template = "html_builder.ButtonOption";
+    static components = { ...defaultBuilderComponents, Button };
+    static props = {};
+
+    setup() {
+        this.isActiveItem = useIsActiveItem();
+        this.state = useDomState((editingElement) => {
+            const isUnremovableButton = editingElement.matches("a.btn.oe_unremovable");
+            return {
+                forceDuplicateButton: !isUnremovableButton,
+                forceNoDeleteButton: isUnremovableButton,
```

Comment:
forceDuplicateButton and forceNoDeleteButton don t exist anymore. Could you check with SOBO ? She is the expert :)